src/NServiceBus.Core/Extensibility/ContextBag.cs

```diff
@@ -143,8 +144,15 @@ internal void Merge(ContextBag context)
             }
         }
 
+        internal IBehavior[] Behaviors
```

Comment:
Should I add a comment here that explains this monster?

Reply:
Definitely.

Reply:
Done